WcaOnRails/app/webpacker/components/RolesTab/RoleForm.jsx

```diff
@@ -23,19 +25,58 @@ const delegateStatusOptions = ['trainee_delegate', 'candidate_delegate', 'delega
 
 export default function RoleForm({ userId, isActiveRole }) {
   const { data, loading, error } = useLoadedData(roleDataUrl(userId, isActiveRole));
+  const regionsFetching = useLoadedData(apiV0Urls.userGroups.list(groupTypes.delegate_regions));
```

Comment:
Why are you not destructuring this into `data, loading, error` like you do with the role data directly above?

Reply:
TBH, I was lazy to do it 😁 Did it now.